test/Feature/ResourceArrays/array-global.test

```diff
@@ -50,6 +50,15 @@ Buffers:
     ArraySize: 4
     ZeroInitSize: 32
 
+  - Name: ExpectedBufB
```

Comment:
This test has been added as part of the resource arrays support, and it was intentionally testing `BufB` results via FileCheck to make sure we are printing arrays correctly.